daffodil-cli/src/main/scala/org/apache/daffodil/cli/Main.scala

```diff
@@ -140,7 +140,10 @@ class CLIConf(arguments: Array[String], stdout: PrintStream, stderr: PrintStream
         s match {
           case Nil => Right(None) // flag was not present
           case (_, Nil) :: Nil => Right(Some(None)) // flag was present but had no parameter
-          case (_, v :: Nil) :: Nil => { // flag was present with a parameter, convert the parameter
+          case (
+                _,
+                v :: Nil
+              ) :: Nil => { // flag was present with a parameter, convert the parameter
```

Comment:
Curious why some lines get expanded like this and others get compressed onto a single line.

Reply:
I can't really say. I imagine this has to do with long lines and splitting function parameters per line.

We really shouldn't disable scalafmt, so if we don't like these changes, we either need to find and modify a config option that changes it (not sure what that would be or if it would affect other places), open a bug with scalafmt if we think it's wrong, or rewrite it in some way that doesn't require this formatting. Of those, the last is probably the best option, but I'm not sure how much it's wroth it.

I generally let scalafmt do its thing for existing code and just focus on ensuring new code is formatted reasonably.

Reply:
The avoidForSimpleOverflow=[slc] is definitely a reasonable improvement. Loosening line length restrictions in favor a readability is usually a win. That said, in most of the changes I would think just moving the comment to its own line is probably more readable, but I'm not sure it's worth the effort to fix.